### PR TITLE
Ajc optimus wrapper

### PR DESCRIPTION
**Pull request by ambrosejcarr**
@samanehsan 

PR not for merging. 

Here's the branch with the adapter WDL for Optimus.  You can see the place on line 101 in `adapter.wdl` where I've indicated that `submit` needs to be called. 

As we discussed, you'll also need a translation wdl that gets called somewhere.

### CODE REVIEW

**Comment by samanehsan** [comment]
Is the failing jenkins build related to these changes?

File: adapter_pipelines/Optimus/adapter.wdl [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+task GetInputs {
+  String bundle_uuid
+  String bundle_version
+  String dss_url
+  Int retry_seconds
+  Int timeout_seconds
+
+  command <<<
+    python <<CODE
+    from pipeline_tools import utils
```
Comment (rexwangcc) [inline adapter_pipelines/Optimus/adapter.wdl]: `from pipeline_tools import dcp_utils` if you want to use `humancellatlas/pipeline-tools:0.1.5`

File: adapter_pipelines/Optimus/adapter.wdl [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+    print('Getting bundle manifest for id {0}, version {1}'.format(uuid, version))
+    manifest_files = utils.get_manifest_files(uuid, version, dss_url, timeout_seconds, retry_seconds)
+
+    print('Downloading assay.json')
+    assay_json_uuid = manifest_files['name_to_meta']['assay.json']['uuid']
+    assay_json = utils.get_file_by_uuid(assay_json_uuid, dss_url)
+
+    # Parse inputs from assay_json and write to fastq_inputs
+    print('Writing fastq inputs to fastq_inputs.tsv')
+    lanes = assay_json['seq']['lanes']
```
Comment (rexwangcc) [inline adapter_pipelines/Optimus/adapter.wdl]: In v4.5.0 metadata schema, this has been changed to `assay_json[content]['seq']['lanes']`

**Comment by samanehsan** [comment]
Continued in #7

**Pull request closed without merging**